batch/2.1/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Jakarta Batch 2.1 (Under Development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+
+Jakarta Batch specifies a Java API plus an XML-based job specification language (JSL), which lets you compose batch jobs in XML from reusable Java application artifacts and conveniently parameterize different executions of a single job.
+
+# Plan
+Jakarta Batch 2.1 will target the Jakarta EE 10 platform release.
+
+Jakarta Batch 2.1 will be a minor update with fixes and enhancements.
+
+## It will contain:
+
+* A list of key or headline enhancements.  This set is being discussed, prioritized via the Jakarta Batch dev mailing list and tracked at: https://github.com/eclipse-ee4j/batch-api/milestone/1
+
+  Note that at the current pace it is likely that the 2.1 delivery will include less than the full list of issues currently grouped within this milestone (which is more of a draft proposal than firm plan).
+
+* Other minor fixes/enhancements to the Jakarta Batch API from our backlog at: https://github.com/eclipse-ee4j/batch-api/issues
+
+  This is the "everything else" category that wouldn't necessarily stand out in a release summary except as "other minor enhancements".  A good deal of the prioritization among these will likely come from the choice of individual committers deciding what to work on.
+
+### It may contain (depending on resources):
+
+* Completion of the EE platform portion of the Batch standalone TCK, so that an EE implementation can simply validate Batch compliance against the standlone TCK.  This would allow the Jakarta Batch project to avoid having to maintain a duplicate "fork" of these tests.
+
+### It will also contain:
+
+* Bug fixes as they arise during the release cycle
+* Any updates required to meet the Java version requirements of Jakarta EE 10
+* Any requirements identified by other specifications or the Jakarta EE 10 platform projects during the release cycle
+
+## Optional Features: 
+
+The integration with CDI deserves some discussion.
+
+The high-level intent is to say that when CDI is present, as within the Jakarta EE Platform, then the integration of Jakarta Batch + CDI will be well-defined and so required for an implementation to be certified as Jakarta Batch-compliant.   However, when CDI is not present, an implementation should be able to certify full Jakarta Batch compliance.
+
+TBD how to phrase this practically, so that it can be implemented by the TCK, for example.   E.g. should this be an optional subset of tests within this TCK or should the relevant tests no-op or behave differently if/when CDI is not detected, (e.g. because `CDI.current()` returns null).
```

Comment:
We want to move away from optional behavior, so it should be phrased as a requirement for when running batch in CDI aware environments and the TCK can validate that using something like the CDI.current() check you mention.

Reply:
I like that approach better.   Made the change.